site/content/post/euler-6-proposal.md

```diff
@@ -0,0 +1,74 @@
+
+We also found an issue with the consumption of bandwidth during linking. It was produced during last-minute featuring before Euler-5 launch. The issue leads to fast bandwidth regeneration as if it was consumed for linking.
+
+All of this is now solved, and plenty of other new features are introduced in the new release!
+
+## State migration proposal by @SaveTheAles
+To update the chain me propose the following state migration (read the [full proposal here](https://github.com/cybercongress/launch-kit/blob/0.1.0/migrations/e-6%20migration_proposal.md)).
+
+4 major things should be taken into account:
+- Cyberlinks made by an agent
+- Karma of an agent
+- Precommits by a validator
+- The stake of a validator
+
+Given the circumstances, an agents' karma was compromised by the bug described by @litvintech, so there is no sense to migrate karma values between testnets.
+
+The solution is in migrating cyberlinks. It is possible to export all of the cyberlinks made by an agent to an IPLD-like structure and add an opportunity for the agents to make their cyberlinks again in the new testnet, with one single transaction. In other words: in the new testnet, the agents can get their links from the previous testnet by a query to an IPLD structure, generate a transaction with all their previously made cyberlinks, sign it and broadcast it to the new network using their ledger via the [cyber.page app](https://cyber.page/pocket) or manually, via CLI.
+
+It would also be cool to create a small guide with an additional script for transaction generation and signing (UNSURE WHY WE NEED THIS SENTENCE).
+
+This lets us cover two of the migration points: cyberlinks and karma. The weakness of that method is that anyone can relink all the previously made cyberlinks before the original discoverers.
+
+The migration of the precommits is complex. To avoid any additional changes in the distribution, we propose to save the state of the precommits in [a .csv file](https://github.com/cybercongress/launch-kit/tree/0.1.0/game_rewards_calculations) at a certain block height and use it for the Game of Links recalculations after the Game. Thus, we can avoid major changes in the distribution and make the migration a smoother process.
+
+The stake of the validators should be burned. The complexity of changing distribution parameters is much higher than the error of the final stake results.
+
+Additionally, we are now able to test the genesis ceremony (LINK - WHAT IS IT?) and parameters discussion. The [launch-kit repo](https://github.com/cybercongress/launch-kit) is ready for newcomers and it seems possible to do this now.
+
+The workflow should be fairly easy:
+1. Announce network halt at a given block height
+2. Export precommits and cyberlinks on a given block
```

Comment:
do we need to export the stake of validators at this stage too?

Reply:
No, we will not do it. As I explained in the doc:

> The stake of the validators should be burned. The complexity of changing distribution parameters is much higher than the error of the final stake results.

This way the accounts part of e-6 genesis file will the same as e-5.